cli/main.c

```diff
@@ -1029,7 +1034,10 @@ static int port_bind_info(int argc, char **argv)
 
 	argconfig_parse(argc, argv, desc, opts, &cfg, sizeof(cfg));
 
-	printf("physical port:%d\n", cfg.phy_port);
+	if (cfg.phy_port == 0xff)
```

Comment:
You've given no way for the user to know about this special un-intuitive 0xff value.  You should make that the default so the user either provides the -f option or not, if they do not it gives all ports.